codecov.yaml

```diff
@@ -0,0 +1,15 @@
+ignore:
+  - "*/test/*"
+  - "crates/wasm/*"
+
+comment:
+  layout: "reach, diff, flags, files"
+  require_changes: true
+
+github_checks:
+  annotations: false
+
+status:
```

Comment:
`status` field is unknown according to codecov.io/validate 

```bash
cat codecov.yaml | curl --data-binary @- https://codecov.io/validate
```

Reply:
@appcypher derp, yep, i forgot coverage.